tests/components/sunsynkweb/conftest.py

```diff
@@ -0,0 +1,101 @@
+"""Common fixtures for the Sunsynk Inverter Web tests."""
```

Comment:
I'd also suggest (while not a hard requirement) to add tests for your `sensors`. This can be easily done in a couple of lines of code with `snapshot` tests. Take a look at the `tedee` sensor test `test_sensors` for an example.

Reply:
all right, so I looked at this and the above(on `test_coordinator`) and have sketched something -- one thing I don't seem to be able to do is get the sensors to update: `SunsynkApiSensor._handle_coordinator_update` is getting called, but `hass.states.get("sensor.solar_battery_charge").state` remains `unknown`; what am I missing?

Reply:
sometimes you need the `freezer` to move the time a bit in the future. Examples in `tedee` `test_lock.py`